app_dev.yaml

```diff
@@ -17,7 +17,17 @@ inbound_services:
 
 default_expiration: "10m"
 handlers:
-# PRODUCTION STATIC
+# Note: The handlers in this file contain two separate sections:
+# DEPLOYMENT SECTION: Required for deployment.
+# NON DEPLOYMENT SECTION: Required for a local setup (both in dev & prod).
+# The comments representing the start & end of these sections should
+# not be modified since they are required for generation of app.yaml.
+# The backend tests to verify this structure: scripts/app_yaml_test.py
```

Comment:
Use complete sentences: The backend tests to verify this structure are located at scripts/app_yaml_test.py.